moped-etl/arcgis/components_to_agol.py

```diff
@@ -195,15 +255,17 @@ def main(args):
         "-d",
         "--date",
         type=str,
+        nargs="?",
+        const=(datetime.now(timezone.utc) - timedelta(days=7)).isoformat(),
         default=None,
-        help=f"ISO date string with TZ offset (ex. 2024-06-28T00:06:16.360805+00:00) of latest updated_at value to find project records to update.",
+        help="ISO date string with TZ offset (ex. 2024-06-28T00:06:16.360805+00:00) of latest updated_at value to find project records to update. Defaults to 7 days ago if -d is used without a value.",
```

Comment:
I gave the `-d` option a default argument of "the last week". This was handy during development and feels like a genuine enhancement, so leave it in?

If you specify the date like usual, the behavior is unchanged of course.